editor/debugger/editor_debugger_tree.cpp

```diff
@@ -62,6 +63,15 @@ void EditorDebuggerTree::_notification(int p_what) {
 			connect("item_collapsed", callable_mp(this, &EditorDebuggerTree::_scene_tree_folded));
 			connect("item_mouse_selected", callable_mp(this, &EditorDebuggerTree::_scene_tree_rmb_selected));
 		} break;
+
+		case NOTIFICATION_THEME_CHANGED: {
+			const StringName icon_max_width = "icon_max_width";
+			const int class_icon_size = get_theme_constant("class_icon_size", EditorStringName(Editor));
+			if (has_theme_constant_override(icon_max_width) && get_theme_constant(icon_max_width) == class_icon_size) {
```

Comment:
I'm not sure if this is needed? At least we never did that on other locations where we clamp the icon size of global classes.

Reply:
You can refer this:
https://github.com/godotengine/godot/blob/bd2300d77a6008167043f23fd91bcc562cde0a19/editor/gui/scene_tree_editor.cpp#L931-L932

The different between this pr and `SceneTreeEditor` is that this pr only set them constant override if changed.
Otherwise, it will fall into infinite loop ( already tested).